requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 mock
-pytest
+pytest!=3.3.0
```

Comment:
Travis was loading 3.3.0, which has a bug that was causing tests to fail:
https://github.com/pytest-dev/pytest/pull/2958